docker/run_celery.bash

```diff
@@ -1,10 +1,10 @@
-#!/usr/bin/env bash
+#!/bin/bash
 set -e
-
 source /etc/profile
 
-CELERYD_OPTIONS="-Ofair --beat --loglevel=DEBUG"
-
+# Run the main Celery worker
 cd "${KOBOCAT_SRC_DIR}"
-
-exec /sbin/setuser "${UWSGI_USER}" python manage.py celeryd $CELERYD_OPTIONS
+exec celery worker -A onadata -Ofair --loglevel=info \
```

Comment:
Does this run Celery as `root` instead of `wsgi`, and if so, is that what you want?

Reply:
Right! Actually I used `kpi` file as a template. Because `kpi` is newer, I thought that's they way we wanted it to be.